spec/models/season/phase_switcher_spec.rb

```diff
@@ -26,9 +25,11 @@
   end
 
   describe '#fake_application_phase' do
+
     it 'timeshifts the application phase to today' do
+      phase = :fake_application_phase
       Timecop.travel(Season.current.applications_close_at - 1.month) do
-        Season::PhaseSwitcher.fake_application_phase
+        Season::PhaseSwitcher.destined(phase)
         expect(subject).to be_application_period
       end
```

Comment:
Yes and no. The describe block says you're testing the actual `fake_application_phase`, but that one's now private API. You shouldn't test private methods. It would stick with the changed method call but would update the describe block to a `context` in `describe '#destined' do … end` with something human readable